README.md

```diff
@@ -24,7 +24,44 @@ __Note:__ On Linux and MacOS, you will need to enable executable permission befo
 
 _e.g._ `$ chmod u+x fpm-v0.1.0-linux-x86_64`
 
-### Build from source
+For other platforms and architectures have a look at the [bootstrapping instructions](#bootstrapping-fpm).
+
+### Creating a new project
+
+Creating a new *fpm* project is as simple as running the command
+`fpm new project_name`. This will create a new folder in your current directory
+with the following contents and initialized as a git repository.
+
+* `fpm.toml` – with your project’s name and some default standard meta-data
+* `README.md` – with your project’s name
+* `.gitignore`
+* `src/project_name.f90` – with a simple hello world subroutine
+* `app/main.f90` (if `--with-executable` flag used) – a program that calls the subroutine
+* `test/main.f90` (if `--with-test` flag used) – an empty test program
+
+### Building your Fortran project with fpm
+
+*fpm* understands the basic commands:
+
+* `fpm build` – build your library, executables and tests
+* `fpm run` – run executables
+* `fpm test` – run tests
+
+The command `fpm run` can optionally accept the name of the specific executable
+to run, as can `fpm test`; like `fpm run specific_executable`. Command line
+arguments can also be passed to the executable(s) or test(s) with the option
+`--args "some arguments"`.
+
+See additional instructions in the [Packaging guide](PACKAGING.md) or
+the [manifest reference](manifest-reference.md).
+
+<details>
+<summary><b>Bootstrapping instructions</b></summary>
+
+### Bootstrapping instructions
```

Comment:
Is it worth mentioning the `install.sh` script we have here as well?

Reply:
Good idea. If `install.sh` indeed does what those instructions say (I used it successfully a few times on Linux, but not on other OSs), then we should replace the stack instructions with just "Run `./install.sh`". That was the whole point of the script.

Reply:
Done.